pages/index.js

```diff
@@ -182,7 +107,107 @@ export default function Home() {
                   </Flex> */}
                 </TableContainer>
               </CardBody>
-            </Card>}
+            </Card>
+}
+
+function useCityData() {
+  const [itemsPerPage, setItemsPerPage] = useState(25);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [data, setData] = useState([]);
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  const getData = useCallback(async () => {
+    try {
+      // docs: https://dev.socrata.com/foundry/data.cityoforlando.net/5pzm-dn5w
+      const apiUrl = `https://data.cityoforlando.net/resource/5pzm-dn5w.json?application_type=Building Permit`
+      // const apiUrl =`https://data.cityoforlando.net/resource/5pzm-dn5w.json?parcel_number=312431779301390`;
+      // const apiUrl = `https://data.cityoforlando.net/resource/5pzm-dn5w.json?$limit=${itemsPerPage}&$offset=${currentPage}`
+      const response = await fetch(apiUrl)
+      const data = await response.json()
+      console.log(data)
+      setData(data)
+    } catch (error) {
+      console.log(error)
+    } finally {
+      setIsLoading(false)
+    }
+  }, [setData])
+
+  const lotData = useMemo(() => {
+    const lots = data.reduce((acc, permit) => {
+      const lotNumber = Number(permit.parcel_number.slice(-5, -1))
+      if (acc.hasOwnProperty(lotNumber)){
+        acc[lotNumber].permits.push(permit)
+      } else {
+        acc[lotNumber] = {
+          number: lotNumber,
+          address: permit.permit_address,
+          // we don't actually need this
```

Comment:
The permit?

Reply:
we don't actually use the `permits` array that I'm creating here on the homepage, but created it anyway 🤷